frontend/src/store/controller/selectors.ts

```diff
@@ -196,9 +197,50 @@ export const getAlarmLimitsRequest = createSelector(
   getAlarmLimits,
   (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.request,
 );
-export const getAlarmLimitsRequestStandby = createSelector(
+export const getAlarmLimitsRequestDraft = createSelector(
   getAlarmLimits,
-  (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.standby,
+  (alarmLimits: AlarmLimitsRequestResponse): AlarmLimitsRequest | null => alarmLimits.draft,
+);
+const alarmLimitsCurrentSelector = (key: string) =>
```

Comment:
I'd suggest renaming this to `rangeSelector` and making it similar to `numericParameterSelector` above: i.e. it takes two arguments, the first is a selector which would be either `getAlarmLimitsCurrent`, `getAlarmLimitsRequest`, or `getAlarmLimitsRequestDraft`, and the second is the key. That way we only need to define one function which is used to make 6 selectors for SpO2 and HR.